### PR TITLE
Add username field and change populate.go to add it

### PR DESCRIPTION
**Pull request by seubert**
* Add username field
* Add migrations
* Instead of checking for existence, do an upsert based on the email field in populate.go

### CODE REVIEW

File: augit/models/github_user.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ type GithubUser struct {
 	CreatedAt time.Time `json:"created_at" db:"created_at"`
 	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
 	Email     string    `json:"email" db:"email"`
+	Username  string    `json:"username" db:"username"`
```
Comment (maxgardner) [inline augit/models/github_user.go]: Below this, with the Validate fn, should we add an additional check that the username field is present?

File: cmd/populate.go [hunk 2]
```diff
@@ -69,14 +56,38 @@ func (adb *AugitDB) checkForDeletion(inUser *swio.User) error {
...
+				Username: inUser.Username,
+			}
+			vErrs, err_ := tx.ValidateAndCreate(ghUser)
+			if vErrs.HasAny() {
+				return vErrs
+			}
+			if err_ != nil {
+				return err_
+			}
+		} else if err == nil {
```
Comment (maxgardner) [inline cmd/populate.go]: Can this just be an else clause without the err == nil specification?

Comment (seubert) [reply cmd/populate.go]: yep, good call

File: augit/models/github_user.go (hunk 1 above)
Comment (seubert) [reply augit/models/github_user.go]: Gonna add this after we do the initial population so that we don't have Validate break existing users in the meantime

**Pull request merged**